@xen-orchestra/web-core/lib/components/ui/table-actions/UiTableActions.vue

```diff
@@ -0,0 +1,27 @@
+<template>
```

Comment:
Remember to add the component's version number:
```suggestion
<!-- v2 -->
<template>
```
Or, since we don’t handle the mobile version yet, you can use `<!-- WIP -->` for example.

Reply:
It was one of my first components, so when I came back to it for the push, I forgot this part 😨